R/loopy.R

```diff
@@ -0,0 +1,286 @@
+loopy <- function(resamples, grid, static) {
```

Comment:
We exported this in melodie, but that was for convenience. Any reason to export it?

Reply:
I will make that change but do it in the next PR after this is merged.